libs/coding/string_utf8_multilang.hpp

```diff
@@ -52,12 +52,15 @@ class StringUtf8Multilang
 public:
   struct Lang
   {
+    static constexpr size_t kMaxTransliterators = 2;
+    using TransliteratorsList = std::array<std::string_view, kMaxTransliterators>;
+
     /// OSM language code (e.g. for name:en it's "en" part).
     std::string_view m_code;
     /// Native language name.
     std::string_view m_name;
     /// Transliterators to latin ids.
-    std::vector<std::string_view> m_transliteratorsIds;
+    TransliteratorsList m_transliteratorsIds;
```

Comment:
Did you check usage?
- Before: m_transliteratorsIds.size() == 1, rare 2.
- After: m_transliteratorsIds.size() == 2 always.